### ISSUE TITLE
🐛 [Bug]Exodus quest

### ISSUE THREAD

**Issue by pak3935** [alexkulya/pandaria_5.4.8#97]
https://www.wowhead.com/quest=24438/exodus
Когда садишься на делижанс, нпс "Оглобля" уезжает, а ты вместе с повозкой остается стоять на прежнем месте.
Если слезть с повозки тебя переносит в то место где сейчас бежит нпс "Оглобля" и ты так же остаешься на месте, а нпс бежит вперед по маршруту.
https://i.yapx.ru/Vefjy.png
https://i.yapx.ru/Vefj0.png

**Comment by pak3935** [comment]
Для себя сделал пока так, с транспортом не умею работать... (
`            if (Creature* harness = player->FindNearestCreature(NPC_STAGECOACH_HARNESS, 30.0f, true))
             //  player->SummonCreature(NPC_HARNESS_SUMMONED, harness->GetPositionX(), harness->GetPositionY(), harness->GetPositionZ(), harness->GetOrientation(), TEMPSUMMON_MANUAL_DESPAWN, 600000, const_cast<SummonPropertiesEntry*>(sSummonPropertiesStore.LookupEntry(3105)));
				player->NearTeleportTo(-2216.45f, 1810.68f, 11.8f, 3.0f);`

**Comment by pak3935** [comment]
у себя сделал пока временно тп сразу на ласт точку, что бы без проблемы было проходимо, как с транспортом решить вопрос пока не знаю 
```
 if (Creature* harness = player->FindNearestCreature(NPC_STAGECOACH_HARNESS, 30.0f, true))
              //  player->SummonCreature(NPC_HARNESS_SUMMONED, harness->GetPositionX(), harness->GetPositionY(), harness->GetPositionZ(), harness->GetOrientation(), TEMPSUMMON_MANUAL_DESPAWN, 600000, const_cast<SummonPropertiesEntry*>(sSummonPropertiesStore.LookupEntry(3105)));
				player->NearTeleportTo(-2216.45f, 1810.68f, 11.8f, 3.0f);
```

**Comment by alexkulya** [comment]
Да, я это заметил. В данный момент занимаюсь исправлением этого задания.
